tests/instruments/test_all_instruments.py

```diff
@@ -0,0 +1,88 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+
+from inspect import isclass
+
+import pytest
+
+from pymeasure.adapters import FakeAdapter
+from pymeasure import instruments
+
+
+class Adapter(FakeAdapter):
```

Comment:
Minor nitpick: please don't name this class `Adapter` as it's identical to our adapter base class. `TestAdapter`?
I also later realised, we don't care what the adapter does in this test, at all, I think, so we might as well use a `MagicMock` instead of defining our own skeleton class.

Reply:
A consequence of being tired and wanting to finish...

Reply:
Hey, I understand you -- I still have 10 unread pymeasure notifications (all PR updates)... not today... :-)